banned-dependencies.lst

```diff
@@ -0,0 +1,32 @@
+# Define each banned dependency as "<groupid>:<artifactid>;<alternative-groupid>:<alternative-artifacid>[;<comment>]"
+org.apache.commons:commons-lang3;io.jenkins.plugins:commons-lang3-api
+org.apache.commons:commons-lang;io.jenkins.plugins:commons-lang3-api
+org.apache.commons:commons-text;io.jenkins.plugins:commons-text-api
+org.apache.commons:commons-compress;io.jenkins.plugins:commons-compress-api
+org.apache.commons:commons-math3;io.jenkins.plugins:commons-math3-api
+
+org.json:json;io.jenkins.plugins:json-api
+com.jayway.jsonpath:json-path;io.jenkins.plugins:json-path-api
+com.google.code.gson:gson;io.jenkins.plugins:gson-api
+com.fasterxml.jackson.core:jackson-core;org.jenkins-ci.plugins:jackson2-api
```

Comment:
My assumption is that jackson-core is always included.

Reply:
It can often come as a dependency of another Jackson one but we can see how it goes

Reply:
Another idea would be to ban the complete groupIds, e.g. `com.fasterxml.jackson.core` , `com.fasterxml.jackson.datatype`, `com.fasterxml.jackson.module` and `com.fasterxml.jackson.dataformat`